doc/source/report/plots/plot_distribution.py

```diff
@@ -0,0 +1,14 @@
+import matplotlib.pyplot as plt
+from mousestyles. import data
+from mousestypes.est_power_param import fit()
```

Comment:
I assume this is a typo: `mousestypes`?
Also, when you import functions, you shouldn't have `()`.